<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 57, 102);
</string>
  </property>
  <widget class="QTableWidget" name="tableContatos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>861</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
font: 75 15pt &quot;Segoe UI Bold&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Telephone</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnGerarPDF">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 57, 102);
color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Get PDF</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlterar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 57, 102);
color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcluir">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 57, 102);
color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Exclude</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVoltar">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 57, 102);
color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
